src/liboslexec/oslexec_pvt.h

```diff
@@ -1057,6 +1063,10 @@ class ShaderInstance {
         // lazyunconnected is on.
         if (!outgoing_connections() && !shadingsys().m_lazyunconnected)
             return false;
+        // Shaders without any downstream connections are not lazy unless
+        // lazyunconnected is on.
```

Comment:
I realize this comment is a cut and paste error and doesn't match the code. Will fix momentarily...